example/tests/test_solid_urls.py

```diff
@@ -133,6 +133,23 @@ def test_home_page_default_prefix_ru_404(self):
 
     # settings
 
+    @override_settings(SOLID_I18N_PREFIX_STRICT=False)
+    def test_about_page_strict_prefix(self):
+        response = self.client.get('/ru/about/')
+        self._base_page_check(response, "ru", "about")
+        self.assertContains(response, '<test>/ru/about/</test>')
+        response = self.client.get('/ru-slug/about/')
+        self._base_page_check(response, "ru", "about")
+        self.assertContains(response, '<test>/ru-slug/about/</test>')
+
+    @override_settings(SOLID_I18N_PREFIX_STRICT=True)
+    def test_about_page_strict_prefix(self):
```

Comment:
Unfortunately, this test has the same name as the previous one. As a result, the first test (with `SOLID_I18N_PREFIX_STRICT=False`) will never run.
Propose to use following names:
`test_about_page_strict_prefix_false`
`test_about_page_strict_prefix_true`

And after I've renamed the test, the first one fails (it returns 404).